lisa/tools/modprobe.py

```diff
@@ -20,13 +20,16 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
     # to leave the node without a network connection if things go wrong.
     def _reload_hv_netvsc(self) -> None:
         # These commands must be sent together, bundle them up as one line
+        # If the VM is disconnected after running below command, wait 60s is enough.
+        # Don't need to wait the default timeout 600s. So set timeout 60.
         self.node.execute(
             "modprobe -r hv_netvsc; modprobe hv_netvsc; "
             "ip link set eth0 down; ip link set eth0 up;"
             "dhclient -r eth0; dhclient eth0",
             sudo=True,
             shell=True,
             nohup=True,
+            timeout=60,
```

Comment:
do you see timeout case? I so nohup is used, it should return immediately

Reply:
Yeah, I saw it returned after 10mins one time.